src/i18n/en/components.json

```diff
@@ -56,6 +56,15 @@
     "kills": "Kills",
     "deaths": "Deaths"
   },
+  "leaderboardGuild": {
+    "weeklyTitle": "Weekly Leaderboard",
+    "monthlyTitle": "Monthly Leaderboard",
+    "title": "Leaderboard",
+    "rank": "#",
+    "guildName": "Name",
```

Comment:
I think this should probably say "Guild" rather than "Name".

But looks good to me, so will merge and we can change that after.